src/intersection-observer.ts

```diff
@@ -67,7 +71,7 @@ export class SpanielIntersectionObserver implements IntersectionObserver {
   private id: string;
   private scheduler: ElementScheduler;
   private callback: Function;
-
+  public parent: SpanielInstance;
```

Comment:
No, It must be private. I will make the change. Even in spaniel-observer I have to make the same change